src/utils/base.test.js

```diff
@@ -0,0 +1,221 @@
+import { getStyles } from './base';
+/* eslint-disable */
+describe('getStyles', () => {
+  let fn;
```

Comment:
I know I was the one who demo'ed using `fn` and `res` 🤦‍♂️ , but let's use variable names more consistent with what the function actually does. `res` in particular has significant meaning on the server-side. What do you think about this:

~~`fn`~~ -> `generateStyles`
~~`res`~~ -> `styles`